app/src/main/java/com/jerboa/ui/theme/Sizes.kt

```diff
@@ -22,8 +22,11 @@ val THUMBNAIL_CARET_SIZE = 10.dp
 
 val DRAWER_ITEM_SPACING = 24.dp
 
-// TODO remove all DPs from code, put here.
+// TODO this needs to be removed when Intrinsic height min is fixed
+// See https://github.com/alorma/Compose-Settings/issues/203
+val SETTINGS_MENU_LINK_HEIGHT = 88.dp
```

Comment:
How was this height chosen?

Arbitrarily based on what fits? Would this still work when a user uses a different fontsize? That is much bigger?

Reply:
Its from [here](https://github.com/alorma/Compose-Settings/blob/main/compose-settings-ui/src/commonMain/kotlin/com/alorma/compose/settings/ui/internal/SettingsTileScaffold.kt#L28).

No need to get too complicated with it. We should be able to remove this hack with the next compose-settings release.